src/Symfony/Bundle/FrameworkBundle/Command/TranslationDebugCommand.php

```diff
@@ -140,6 +140,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             $transPaths[] = $this->defaultTransPath;
         }
         $viewsPaths = $this->viewsPaths;
+        $viewsPaths[] = $kernel->getProjectDir().'/src';
```

Comment:
I would create a new array as `src/` does not contain views, something like `$codePaths`?

Reply:
@fabpot just to provide context, currently if you inject `TranslatorInterface`, those classes get added to `$viewPaths` even though it doesn't contain any templates.

Here's an example dump:

```
^ array:16 [
  0 => "public"
  1 => "/var/www/html/vendor/symfony/twig-bridge/Resources/views/Email"
  2 => "/var/www/html/vendor/symfony/twig-bridge/Resources/views/Form"
  3 => "/var/www/html/vendor/symfony/framework-bundle/Command/TranslationDebugCommand.php"
  4 => "/var/www/html/vendor/symfony/form/Extension/Core/Type/ColorType.php"
  5 => "/var/www/html/vendor/symfony/form/Extension/Core/Type/TransformationFailureExtension.php"
  6 => "/var/www/html/vendor/symfony/form/Extension/Validator/Type/FormTypeValidatorExtension.php"
  7 => "/var/www/html/vendor/symfony/form/Extension/Validator/Type/UploadValidatorExtension.php"
  8 => "/var/www/html/vendor/symfony/form/Extension/Csrf/Type/FormTypeCsrfExtension.php"
  9 => "/var/www/html/vendor/symfony/validator/ValidatorBuilder.php"
  10 => "/var/www/html/vendor/symfony/framework-bundle/CacheWarmer/TranslationsCacheWarmer.php"
  11 => "/var/www/html/vendor/symfony/translation/DataCollector/TranslationDataCollector.php"
  12 => "/var/www/html/src/Controller/SecurityController.php"
  13 => "/var/www/html/vendor/symfony/form/Extension/Core/Type/FileType.php"
  14 => "/var/www/html/vendor/symfony/twig-bridge/Extension/TranslationExtension.php"
  15 => "/var/www/html/vendor/symfony/twig-bridge/Extension/FormExtension.php"
]
```

Can you confirm if we should keep it this way or if we should add `$codePaths` and then if we do add `$codePaths` would we just use it for adding the `src` folder or would we move some of these files into `$codePaths` instead?

Reply:
I would say that the issue is the name `$viewPaths`. When the variable was first introduced, the PhpExtractor was targetting only PHP templates, not other usages of the translator

Reply:
Vars have been renamed to `codePaths`